hoomd/CMakeLists.txt

```
@@ -401,10 +413,9 @@ if (BUILD_METAL AND BUILD_MD)
     add_subdirectory(metal)
 endif()
 
-if (NOT ENABLE_HIP OR HIP_PLATFORM STREQUAL "nvcc")
-    if (BUILD_MPCD)
-        add_subdirectory(mpcd)
-    endif()
+if (BUILD_MPCD AND (NOT ENABLE_HIP OR HIP_PLATFORM STREQUAL "nvcc"))
+    target_compile_definitions(_hoomd PUBLIC ENABLE_MPCD)
```

```suggestion
    target_compile_definitions(_hoomd PUBLIC BUILD_MPCD)
```
Name this definition consistent with the CMake variable that controls it. Also change all instances where this is used (global search and replace may be useful here).

---

Done!